lib/selenium/screenshot.js

```diff
@@ -193,12 +189,6 @@ function getScreenshotPromise(task, cachedDrivers) {
                 driver.wait(untilPageHasLoaded(driver));
             }
 
-            if (task.browser === "phantomjs") {
-                // phantomjs needs some extra time here to render things.
-                // not sure why.
-                driver.sleep(1000);
-            }
```

Comment:
When I said Chromium was faster than Phantom, I'd completely forgotten this hack existed. So it'll be even faster.